packages/template/src/lib/stack-app/apps/implementations/client-app-impl.ts

```
@@ -1193,11 +1198,39 @@ export class _StackClientAppImplIncomplete<HasTokenStore extends boolean, Projec
+    } else {
+      crud = Result.orThrow(await this._customItemCache.getOrWait([session, options.customId, options.itemId], "write-only"));
+    }
+    return this._clientItemFromCrud(crud);
+  }
+
+  // IF_PLATFORM react-like
+  useItem(options: { itemId: string, userId: string } | { itemId: string, teamId: string } | { itemId: string, customId: string }): Item {
+    const session = this._useSession();
+    let crud: ItemCrud['Client']['Read'];
+    if ("userId" in options) {
+      crud = useAsyncCache(this._userItemCache, [session, options.userId, options.itemId] as const, "app.useItem(user)");
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)